config/action.d/abuseipdb.conf

```diff
@@ -102,3 +108,22 @@ actionunban =
 # Register for abuseipdb [https://www.abuseipdb.com], get api key and set below.
 # You will need to set the category in the action call.
 abuseipdb_apikey =
+
+# Option: dst_ips_prefix
+# Notes:  Specifies the prefix used before destination IPs in your log for removing the first two octets of DST IP from match logs.
+# Example: If your iptables log entry is DST=xxx.xxx.xxx.xxx, set dst_ips_prefix to DST=
+# dst_ips_prefix = DST=
+# Values:  STRING  Default: None
+dst_ips_prefix =
+
+# Option: other_removed
+# Notes:  Specifies text to be removed from matches log entries.
+# Example: To remove hostname and app from iptables log entries, set other_removed to HOSTNAME APPNAME:
+# other_removed = myhost kernel:
+# Values:  STRING  Default: None
+other_removed =
+
+# Option: hostname
+# Notes:  Use this option to remove hostname from matches log.
+# Values:  STRING  Default: None
+hostname =
```

Comment:
There are already dynamic action tags for the hostname - see https://github.com/fail2ban/fail2ban/blob/7fd097d73fb995daa39a834879b553a53d1f64ca/fail2ban/server/action.py#L63-L64

So one could use `<fq-hostname>` and/or `<sh-hostname>` instead (and they would not need init definition and should not be supplied as parameters).